reference/5.1/PSReadLine/About/about_PSReadLine.md

```diff
@@ -15,7 +15,12 @@ PowerShell console.
 
 ## Long Description
 
-PSReadLine 2.0 provides a powerful command-line editing experience for the
+PowerShell 5.1 ships with PSReadLine 2.0.0. There are newer versions
+available. The current version of PSReadLine can be installed and used on
+Windows PowerShell 5.1 and newer. For some features, you need to be running
+PowerShell 7.1 or higher.
```

Comment:
Should be 7.2 or higher.

Reply:
I will change the recommendation to 7.2 here.